tools/donate-cpu-server.py

```diff
@@ -490,9 +490,12 @@ def diffMessageIdReport(resultPath: str, messageId: str) -> str:
         if not os.path.isfile(filename):
             continue
         with open(filename, 'rt') as f:
-            diff_stats = f.read()
-        if messageId not in diff_stats:
```

Comment:
This always checked the whole string for the occurrence. This might speed up things a bit if there was a lot of findings - unless it is offset by the added JSON loading.